docs/libcurl/opts/CURLOPT_ALTSVC_CTRL.3

```diff
@@ -69,7 +69,10 @@ Accept alternative services offered over HTTP/3. This will only be used if
 libcurl was also built to actually support HTTP/3, otherwise this bit will be
 ignored.
 .SH DEFAULT
-0. No Alt-Svc treatment.
+Alt-Svc handling is disabled by default. If \fICURLOPT_ALTSVC(3)\fP is set,
+\fCURLOPT_ALTSVC_CTRL(3)\fP has a default value corresponding to CURLALTSVC_H1
```

Comment:
\fCURLOPT_ALTSVC_CTRL(3)\fP the \f should be \fI. Looks fine otherwise.

Reply:
Ack!